client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioning.tsx

```diff
@@ -57,8 +59,9 @@ const ContactQuestioning: React.FC<Props> = ({id}: Props): JSX.Element => {
                     <FormTitle title={`טופס תשאול מגעים (${allContactedInteractions.length})`}/>
                     {
                         allContactedInteractions.sort((firstInteractedContact, secondInteractedContact) =>
-                            firstInteractedContact.phoneNumber ? firstInteractedContact.phoneNumber.localeCompare(secondInteractedContact.phoneNumber) : 0).map((interactedContact) => (
-                            <div key={interactedContact.id} className={classes.form}>
+                            firstInteractedContact.phoneNumber ? firstInteractedContact.phoneNumber.localeCompare(secondInteractedContact.phoneNumber) : 0).map((interactedContact) => {
+                            const isFamilyContact : boolean = isInvolvedThroughFamily(interactedContact.involvementReason);
```

Comment:
Shouldn't you also perform an `isInvolved` check here?

Reply:
Not really, because if it is a family contact it must be involved